broker/broker.go

```diff
@@ -185,6 +185,10 @@ func (b *S3Broker) getBucketNames(instanceNames []string, instanceGUID, serviceG
 		return bucketNames, nil
 	}
 
+	if b.cfClient == nil {
+		return nil, errors.New("unable to get bucket names. you will need to configure a cf_client")
```

Comment:
This error message might be confusing for users. I only just started documenting this in #24, but if `cfClient` is unset, then the operator hasn't configured the broker to support binding to multiple service instances. WDYT about an error message like `This broker is not configured to support binding to additional instances. Contact your Cloud Foundry operator for details.`. And maybe we can put this check in `Bind` so we can fail earlier.

Reply:
Sounds good to me, thanks @jmcarp . Have fixed in that direction and added an additional commit to the PR (note that other than building, I haven't tested further).